groups/infrastructure/main.tf

```diff
@@ -0,0 +1,94 @@
+terraform {
+  required_version = "~> 1.3"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.54.0"
+    }
+    vault = {
+      source  = "hashicorp/vault"
+      version = "~> 3.18.0"
+    }
+  }
+}
+
+provider "aws" {
+  region  = var.aws_region
+}
+
+terraform {
+  backend "s3" {}
+}
+
+module "file_transfer_alb" {
+  source = "git@github.com:companieshouse/terraform-modules//aws/application_load_balancer?ref=1.0.296"
+  count = "${var.file_transfer_create_alb == 1 ? 1 : 0}"
+
+  environment         = var.environment
+  service             = "file-transfer"
+  ssl_certificate_arn = data.aws_acm_certificate.cert.arn
+  subnet_ids          = values(local.routing_subnet_ids)
+  vpc_id              = data.aws_vpc.vpc.id
+  idle_timeout        = 1200
+  route53_domain_name     = var.cert_domain
+
+  create_security_group  = true
+  internal               = true
+  ingress_cidrs          = ["0.0.0.0/0"]
```

Comment:
Ingress cidrs of `0.0.0.0/0` should not be used for internal ALBs. This should be replaced with a suitable list of subnet cidrs specific to the deployment environment and/or suitable prefix list IDs.

This is a security check that will fail if deployed as-is.

Reply:
This is fixed now. Commit is pushed.